NEWS.md

```diff
@@ -0,0 +1,6 @@
+# utime changelog
+
+## 0.2.3
```

Comment:
It'd be good to fix this to 0.3.0 too, although I wouldn't say it needs a new release.

Reply:
Oops, thanks!